.github/.yarnrc

```diff
@@ -0,0 +1 @@
+enableScripts: false
```

Comment:
Do you need this file anymore? It should already be covered under `.yarnrc.yml`.

Reply:
The file is removed